### PR TITLE
Company API route 

### PR DESCRIPTION
**Pull request by MohammedYehia**
- [x] Make a route '/companyinfo/:companyNumber' to get the company information through the company 
  house API https://companieshouse.gov.uk

### CODE REVIEW

File: server/controllers/getCompanyInfo.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const request = require('request');
+require('env2')('./config.env');
+
+module.exports = (req, res)=>{
+    request({
+        url: `https://${process.env.COMPANY_HOUSE_APIKEY}@api.companieshouse.gov.uk/company/${req.params.companyNumber}`
+    }, function (error, response, body) {
```
Comment (sultanassi95) [inline server/controllers/getCompanyInfo.js]: what if there was an `error`, shouldn't we handle it?

Comment (sultanassi95) [reply server/controllers/getCompanyInfo.js]: :tired_face:

Comment (MohammedYehia) [reply server/controllers/getCompanyInfo.js]: oops, you are hundred percent right

File: server/controllers/getCompanyInfo.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const request = require('request');
```
Comment (thesoorae) [inline server/controllers/getCompanyInfo.js]: the standard is to use snake_case for file names, rather than camelCase.

**Pull request merged**